wizeline/project-lab
# Feat - 100 - Moving search field to general layout.

#### What does this PR do?

Search field was moved to general layout.

#### Where should the reviewer start?

Take a look at this new components for search `app/core/components/Search/index.tsx`

#### How should this be manually tested?

On any page search project by name and you should be redirected to the search page with the results.

#### Any background context you want to provide?

I changed the favicon in this PR, I know it's not related to it but I wanted to take advantage of this PR to send it.

#### What are the relevant tickets?

#100 

#### Screenshots

![Screen Shot 2021-11-02 at 10 34 42](https://user-images.githubusercontent.com/47539644/139923046-158446c0-bc42-45fb-9fc6-51e52bde6c87.png)

![Screen Shot 2021-11-02 at 10 34 28](https://user-images.githubusercontent.com/47539644/139923062-258de76e-35ee-44eb-a9cb-65640b5a2937.png)
